yarn-completion.bash

```diff
@@ -921,12 +921,22 @@ _yarn_workspace() {
 			__yarn_count_args
 			case "$args" in
 				[0-2])
-					workspaces_info=$(yarn workspaces info -s 2> /dev/null)
-					if [[ -n $workspaces_info ]]; then
-						mapfile -t < <(
-							sed -n 's/^ \{2\}"\([^"]*\)": {$/\1/p' <<< "$workspaces_info"
-						)
-						COMPREPLY=($(compgen -W "${MAPFILE[*]}" -- "$cur"))
+					if [ -f .yarnrc.yml ]; then
```

Comment:
This will only work properly if you are completing yarn from the root of the project. If you call `yarn workspace <Tab>` from any child directories, this will fail.

See the following snippet for how the script locates the `package.json` file.

https://github.com/dsifford/yarn-completion/blob/73fe6781037336d210ba25b9dd0e90a335839e58/yarn-completion.bash#L35-L41